platform/wm/rhodes/browser/CEBrowserEngine.cpp

```diff
@@ -87,6 +87,40 @@ LRESULT CEBrowserEngine::CreateEngine()
 	DWORD			dwFlags;			/// shdocvw client flags
 	IClassFactory	*ppv;				/// pointer variable to hold WebBrowser interface pointer
 
+
+	TCHAR szPlatform[128];
+	memset(szPlatform, 0, 127 * sizeof(TCHAR));
+	SystemParametersInfo(SPI_GETOEMINFO, 127, szPlatform, 0);
+	TCHAR szSIPValue[100];
+	ZeroMemory(szSIPValue,100);
+
+
+		if((_tcsstr(szPlatform,TEXT("MK4000")) != NULL) || (_tcsstr(szPlatform,TEXT("BigBoardMPA3")) != NULL) || (_tcsstr(szPlatform,TEXT("MC18")) != NULL))
```

Comment:
Just use MK3100 and MK3000... not sure what 'BigBoardMPA3' is but I suspect it is a pre-release BSP